### DISCUSSION TITLE
Problem with blockchain

### DISCUSSION THREAD

**Discussion by artempl88** [Chia-Network/chia-blockchain#17070] [Support]
Hi, I have a problem with Chia (XCH) withdrawal from the exchange Gate.io. I have a TX ID, but I don't see it in the blockchain explorer. The status of transaction is: "Done & Blockchain pending 30 minutes". But I am still waiting for my money. The exchange support tells that it is the blockchain problem. Th TX ID is 0x431c9c1755c2da8dd8bc7cc1dba0a7be49153dee7827b94494209e9c8478ea1d

**Comment by BrandtH22** [comment]
Hey @artempl88 , there have been no stalls or issues with the blockchain. Currently there is high usage of the chain which is delaying transaction that have been submitted with no or insufficient fees. 

I would contact the exchange and have them resubmit the transaction with higher blockchain fees to ensure it is processed in a timely manner.

Let us know if you have any further questions and feel free to join our discord server for more timely and thorough support https://discord.gg/chia

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
